docs/cody/capabilities/supported-models.mdx

```diff
@@ -39,13 +39,21 @@ In addition, Sourcegraph Enterprise customers using GCP Vertex (Google Cloud Pla
 
 Cody uses a set of models for autocomplete which are suited for the low latency use case.
 
-|     **Provider**      |                                         **Model**                                         | **Free** | **Pro** | **Enterprise** |     |     |     |     |
-| :-------------------- | :---------------------------------------------------------------------------------------- | :------- | :------ | :------------- | --- | --- | --- | --- |
-| Fireworks.ai          | [DeepSeek-Coder-V2](https://huggingface.co/deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct)                             | ✅        | ✅       | ✅              |     |     |     |     |
-| Fireworks.ai          | [StarCoder](https://arxiv.org/abs/2305.06161)                                             | -        | -       | ✅              |     |     |     |     |
-| Anthropic             | [claude Instant](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | -        | -       | ✅              |     |     |     |     |
-|                       |                                                                                           |          |         |                |     |     |     |     |
+| **Provider** |                                         **Model**                                         | **Free** | **Pro** | **Enterprise** |     |     |     |     |
+| :----------- | :---------------------------------------------------------------------------------------- | :------- | :------ | :------------- | --- | --- | --- | --- |
+| Fireworks.ai | [DeepSeek-Coder-V2](https://huggingface.co/deepseek-ai/DeepSeek-Coder-V2-Lite-Instruct)   | ✅        | ✅       | ✅              |     |     |     |     |
+| Fireworks.ai | [StarCoder](https://arxiv.org/abs/2305.06161)                                             | -        | -       | ✅              |     |     |     |     |
+| Anthropic    | [claude Instant](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | -        | -       | ✅              |     |     |     |     |
+|              |                                                                                           |          |         |                |     |     |     |     |
 
 <Callout type="note">The default autocomplete model for Cody Free, Pro and Enterprise users is DeepSeek-Coder-V2.</Callout>
 
 <Callout type="note">The DeepSeek model used by Sourcegraph is hosted by Fireworks.ai, and is hosted as a single-tenant service in a US-based data center. For more information see our [Cody FAQ](https://sourcegraph.com/docs/cody/faq#is-any-of-my-data-sent-to-deepseek).</Callout>
+
+## Smart Apply
+
+|    **Model**     | **Free** | **Pro** | **Enterprise** |      |     |     |     |     |
+| :--------------- | :------- | :------ | :------------- | :--- | --- | --- | --- | --- |
+| Qwen 2.5 Coder | ✅        | ✅       | ✅              |      |     |     |     |     |
```

Comment:
needs a provider column as in autocomplete (Fireworks.ai)

Reply:
Done